README.md

```diff
@@ -0,0 +1,17 @@
+<p align="center">
+  <img src="./pictures/bazaar-pay-logo-english.png" alt="Vizdom"/>
+</p>
+
+Payment service usage technical documentation
+
+BazaarPay provides multiple payment solutions For merchants. and SDKs for easy and faster adaptation, including:
```

Comment:
BazaarPay provides merchants with multiple payment solutions, and SDKs for easy and fast adoption, including: